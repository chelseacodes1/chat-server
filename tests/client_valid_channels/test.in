REGISTER user2 hello2 
LOGIN user2 hello2
CHANNELS
CREATE news
CHANNELS
CREATE intros 
CHANNELS
CREATE f1
CHANNELS
JOIN news
JOIN intros
JOIN f1 