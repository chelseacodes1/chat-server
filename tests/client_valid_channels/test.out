RESULT REGISTER 1
RESULT LOGIN 1
RESULT CHANNELS 
RESULT CREATE news 1
RESULT CHANNELS news
RESULT CREATE intros 1
RESULT CHANNELS intros, news
RESULT CREATE f1 1
RESULT CHANNELS f1, intros, news
RESULT JOIN news 1
RESULT JOIN intros 1
RESULT JOIN f1 1
